cookbook/doctrine/registration_form.rst

```diff
@@ -224,8 +223,7 @@ controller for displaying the registration form::
             $form->handleRequest($request);
             if ($form->isSubmitted() && $form->isValid()) {
                 // 3) Encode the password (you could also do this via Doctrine listener)
-                $encoder = $this->get('security.encoder_factory')
-                    ->getEncoder($user);
+                $encoder = $this->get('security.encoder_factory')->getEncoder($user);
                 $password = $encoder->encodePassword($user->getPlainPassword(), $user->getSalt());
```

Comment:
The form handling doesn't work as is as we do not have a property for the plain password in the registration form. I think we should use a DTO class to back the form instead.